modules/installation-vsphere-machines.adoc

```diff
@@ -127,6 +126,7 @@ $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${
 **** Navigate to a virtual machine from the vCenter Server inventory.
 **** On the *Configure* tab, expand *Settings* and select *vApp options.*
 **** Scroll down and under *Properties*, apply the configurations that you just edited.
+**** Note: For the bootstrap node, ignition must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window.  This is due to a restriction in the maximum size of data which may be provided in a vApp property.
```

Comment:
@rvanderp3 does it make sense to combine this info with the sub-bullet several lines up?
>`guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type.

Reply:
yep, I think it would be clarifying to move that there.  Making that edit.